temp.go

```diff
@@ -213,19 +214,22 @@ func handleBotCommands(replyChannel chan ReplyChannel) {
 			replyChannel <- reply
 		case "track":
 			if len(commandArray) < 3 {
-				reply.DisplayTitle = "Sorry, I don't have enough information to make an event for you. try `@togglbot track PROJECT_NAME 9:00AM-5:00PM TASK_DESCRIPTION`"
+				reply.DisplayTitle = "Sorry, I don't have enough information to make an event for you. try `@togglbot track PROJECT_NAME 9:00AM-5:00PM YYYY/MM/DD (or 'today') TASK_DESCRIPTION`"
 				replyChannel <- reply
 				break
 			}
 			projectName := commandArray[1]
 			pid := getProjectWithName(togglApiKey, projectName)
 			timeRange := commandArray[2]
+
+			parsedDate, err := parseDate(commandArray[3])
```

Comment:
Ah actually, i suppose thats not the case since description is optional, disregard.